src/java/picard/analysis/directed/CollectHsMetrics.java

```diff
@@ -45,14 +45,42 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Collects a set of Hybrid Selection specific metrics from an aligned SAM" +
-                "or BAM file. If a reference sequence is provided, AT/GC dropout metrics will " +
-                "be calculated, and the PER_TARGET_COVERAGE option can be used to output GC and " +
-                "mean coverage information for every target.",
-        usageShort = "Collects Hybrid Selection-specific metrics for a SAM or BAM file",
+        usage = CollectHsMetrics.USAGE_SUMMARY + CollectHsMetrics.USAGE_DETAILS,
+        usageShort = CollectHsMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectHsMetrics extends CollectTargetedMetrics<HsMetrics, HsMetricCollector> {
+        static final String USAGE_SUMMARY = "Collects hybrid-selection (HS) specific metrics for a SAM or BAM file.  ";
+        static final String USAGE_DETAILS = "<p> Hybrid-selection (HS) is the most commonly used technique to capture exon-specific sequences for targeted sequencing experiments.  " +
+                "For information on HS theory and practice, please see the following GATK " +
+                "<a href=\"http://www.broadinstitute.org/gatk/guide/article?id=6331\">Dictionary</a> entry.</p> "+
+
+                "<p>This tool requires an aligned SAM or BAM file as well as bait and target interval files.  These interval files can be created from BED files using Picard's " +
```

Comment:
> These interval files can be created from BED files 

This is not super clear. What you mean here is "converted", not created, and you don't specify where the BED files come from in the first place. This is what I would write here:

<p>This tool requires an aligned SAM or BAM file as well as bait and target interval files in Picard interval_list format. You should use the bait and interval files that correspond to the capture kit that was used to generate the capture libraries for sequencing. They can generally be obtained from the manufacturer of the kit. If they are provided in BED format, you can convert the BED files to the Picard interval_list format using the <a href=\"http://broadinstitute.github.io/picard/command-line-overview.html#BedToIntervalList\">BedToInterval</a> tool.</p>